source/playbooks/reusing-and-sharing-playbooks.rst

```diff
@@ -1,20 +0,0 @@
-Reuse and share playbooks
-=========================
-
-.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
-  :start-after: :nosearch:
-
-Playbooks are repeatable workflows and sometimes it’s easier to copy and improve rather than start from scratch. 
```

Comment:
Do you know if there already a way to redirect this old removed page to the new one? 

I've checked and the [old one](https://duckduckgo.com/?q=site%3Adocs.mattermost.com+inurl%3Aplaybooks+reusing&t=chromentp&ia=web) is indexed in google. how do we usually operate in this kind of scenario?

Reply:
Great question, @trilopin! I've set up a redirect for this and this no longer displays in our doc search results. As this is a very recent change it may take a while for Google to update the displayed result but I can confirm that when you click the search result it navigates to the new page.